src/helpers/getValue/getItemsPage.ts

```diff
@@ -0,0 +1,48 @@
+import BigNumber from 'bignumber.js';
+
+interface GetItemsPageType {
+  currentPage: number;
+  itemsPerPage: number;
+  items: any[];
+}
+
+export const getItemsPage = ({
```

Comment:
add basic test

Reply:
The setup for spec testing for multiple helper functions will come at a later date in a different PR